src/kubernetes_configmap_jwt_scan/kubernetes_configmap_jwt_scan/scan.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+import logging
+import jwt
+
+from distutils.dir_util import mkpath
+
+import kubernetes_configmap_jwt_scan.context
+
+from kubernetes_configmap_jwt_scan.config import config
+from lib.k8s_configmap_jwt_scan import is_jwt
```

Comment:
This module offers a simple helper method, unrelated to configmaps or kubernetes. Can we just call it `lib.jwt`?